src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -0,0 +1,59 @@
+package frc.robot.subsystems.elevator;
+
+import static frc.robot.subsystems.elevator.ElevatorConstants.*;
+
+import com.ctre.phoenix6.signals.ControlModeValue;
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class Elevator extends SubsystemBase {
+
+    private static Elevator INSTANCE;
+    private final ElevatorInputs inputs = new ElevatorInputs();
+    private final ElevatorIO io;
+    private final Mechanism2d mechanism2d = new Mechanism2d(MECHANISM_WIDTH, MECHANISM_HEIGHT);
```

Comment:
Should be logged with @AutoLogOutput`.

Reply:
cbf18de77bba92907e2683bd70b825d53635c6a8

Reply:
What I meant is that the mechanism object should be logged as an output of this subsystem. This can be done using the `@AutoLogOutput` annotation.

Reply:
f8cb901acad6aff5da7da1666edcf023b1d85ed9

Reply:
I meant logging the `Mechanism2d` object directly. This will make it possible to visualize in AdvantageScope. You can read more in the documentation: https://github.com/Mechanical-Advantage/AdvantageKit/blob/4c2785d30c61097655b5ad899e200e0c3f8801c3/docs/RECORDING-OUTPUTS.md#mechanism2d

Reply:
7001a6d8aa4eab37110ec957163dec145c658646